packages/vats/src/lib-board.js

```diff
@@ -102,13 +102,13 @@ const initDurableBoardState = (
   const immutable = { prefix, crcDigits };
 
   const lastSequence = BigInt(initSequence);
-  /** @type {MapStore<BoardId, PassableCap>} */
+  /** @type {MapStore<BoardId, RemotableObject>} */
```

Comment:
does `ScalarKey` not work here?

Reply:
it does as input, but as output it is wider than downstream expects.

isn't the Board for giving boardIds to remotable object? I was trying to think of a reason you'd put a Primitive in there and couldn't.

Reply:
True, there's no practical value to put a primitive on the board. Documenting the interface as only taking remotables as values is probably fine.

But in that case, why bother with `ScalarKey` at all?

Reply:
exactly. this doesn't use it. It has the import just so the jsdoc block isn't collapsed. I'll remove it.